### ISSUE TITLE
CTA / Organization / Share

### ISSUE THREAD

**Issue by paulsonder** [OpenConceptLab/ocl_issues#2022]

**Comment by paulsonder** [comment]
@jamlung-ri I think this may have been missed. It may be interesting to consider how a user can share an Organization.

**Some ideas:**

1. Share URL by email (or maybe other platforms, such as Whatsapp)
2. Share to a social media platform, such as a LinkedIn
3. Generate some kind of preview (like the card your might generate from OpenGraph information

We should start simply, but some thinking about how and our users wish to share owners and repo's from OCL would be good.

**Comment by jamlung-ri** [comment]
Agreed, and I believe the MVP for this is simply a "Copy URL to Clipboard" for now. Your ideas above are definitely interesting ones though, and I agree to start simply. 

Is that MVP that I wrote above adequate, in your mind?

**Comment by paulsonder** [comment]
Yes, seems very much the MVP and leave some room to iterate post-beta. Thanks @jamlung-ri!
